.github/workflows/tests.yml

```diff
@@ -8,4 +8,19 @@ jobs:
     - uses: actions/setup-ruby@v1
       with:
         ruby-version: '2.6'
-    - run: ruby tests/well-formed-yaml.rb
+    - uses: actions/cache@v1
```

Comment:
Do we need to bother with gem caching? There's only one gem.

Reply:
For now, but also _speeeed_